lib/maths/CBoostedTreeImpl.cc

```diff
@@ -978,6 +977,10 @@ CBoostedTreeImpl::updateForest(core::CDataFrame& frame,
         const auto& treeToRetrain = m_BestForest[index];
         const auto& treeWhichWasRetrained = retrainedTrees.back();
 
+        double eta{index < m_BestForest.size()
```

Comment:
I want to anticipate the case that we consider adding trees as well. Although currently we only ever use `m_RetrainedTreeEta`.